src/plugins/intel_cpu/src/compiled_model.cpp

```diff
@@ -73,6 +77,16 @@ CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
     std::vector<Task> tasks;
     tasks.resize(streams);
     m_graphs.resize(streams);
+    auto initScheduler = [&]() {
+#if defined(OV_CPU_WITH_ACL)
+        static std::once_flag flag_once;
+        std::shared_ptr<arm_compute::IScheduler> acl_scheduler = std::make_shared<ACLScheduler>();
```

Comment:
What is the idea behind creating a new `acl_scheduler` while it's set only once? Why can't we instance it inside the lambda passed to `std::call_once`?